### DISCUSSION TITLE
[M-Z] (15 Gennaio 2018) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#426] [Esami & Esoneri M-Z]
## Traccia

<img width="929" alt="image" src="https://github.com/user-attachments/assets/f2310705-2399-4d1e-8a84-239ac8bd4e26">

> Questo esercizio è stato proposto **anche** come **Esercizio 4** della traccia di esame **del 22 gennaio 2019**

**Comment by rimaout** [comment]
<img width="502" alt="image" src="https://github.com/user-attachments/assets/f58fe990-149a-4c10-be8e-f1b1dbc16e4d">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [*20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/f8afb456-1f93-4612-8263-cee7bfe5debd)

\* _Questo esercizio era in comune rispetto alla traccia del 22 gennaio 2019, gli esercizi 7, 8 e 9 sono specifici delle due rispettive tracce, per quelli vedi [20180115.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20180115.pdf)_
